src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -293,11 +290,9 @@ const StoryPage = ({ pageData }) => {
       />
       <ComscoreAnalytics />
       <NielsenAnalytics />
-      {/* dotcom and dotcomConfig need to be setup before the main dotcom javascript file is loaded */}
-      {isAdsEnabled && !isAmp && (
-        <CanonicalAdBootstrapJs adcampaign={adcampaign} />
+      {allowAdvertising && (
+        <AdContainer slotType="leaderboard" adcampaign={adcampaign} />
```

Comment:
With the `AdContainer` including the `AdBootstrapJs`, will there be a problem rendering 2 of the bootstrap script tags if the MPU block and Leaderboard components are on the same page? Or maybe react-helmet is clever enough to prevent duplicate script tags?

Reply:
That is a very good question! I will check and let you know.

Reply:
I've checked on our preview environment, and I can only see one occurence of the script in the head of the html:

![image](https://github.com/bbc/simorgh/assets/58214768/b1f9bc35-9f8e-4de6-9ab2-902601cff148)

So I think(?) we should be good? 🤞